tutorials/easydeploy-vault/index.mdx

```diff
@@ -0,0 +1,175 @@
+- A valid [API key](/identity-and-access-management/iam/how-to/create-api-keys/)
+- Created a Scaleway Kubernetes [Kapsule](/containers/kubernetes/how-to/create-cluster/) or [Kosmos](/containers/kubernetes/how-to/create-kosmos-cluster/) cluster
+
+## Deploying the Vault application using Easy Deploy
+
+1. In the [Scaleway console](https://console.scaleway.com/), navigate to the **Kubernetes** section under **Containers**.
+2. Click the name of the cluster where you wish to deploy Grafana. The **Cluster Information** tab will display.
+3. Click the **Easy Deploy** tab. The application dashboard displays.
+4. Click **Deploy Application**. The application deployment wizard displays.
+5. Choose **Application Library** to see the list of available applications.
+6. Select the **Grafana** application.
+    <Message type="tip">
+        If you cannot find Vault on the first page, use the search bar or navigate through the library.
+    </Message>
+7. Optionally, customize the default configuration for Grafana using [Helm Charts](/tutorials/kubernetes-package-management-helm/). If you do not require any customized configuration you can skip this step.
+8. Enter a name and a Kubernetes namespace for your application. If no name is entered, Grafana will be installed in the default namespace of the cluster.
+9. Click **Deploy Application** to deploy Grafana on your Kubernetes cluster.
+
+## Initializing and unsealing Vault
+
+1. Check the status of your Vault using the `kubectl` command.
+    ```sh
+    kubectl get pods -l app.kubernetes.io/name=vault
+    ```
+    <Message type="tip">
+      If you choose anothter name for your Vault application ensure to replace the application name with the corresponding value.
+    </Message>
+
+2. Initialize Vault:
+    ```sh
+    kubectl exec -it vault-0 -- vault operator init
+    ```
+    <Message type="important">
+      Save the unseal keys and the initial root token provided by the command.
+    </Message>
+
+2. Unseal Vault using three unseal keys retrieved in the previous step:
+    ```sh
+    kubectl exec -it vault-0 -- vault operator unseal <unseal-key-1>
+    kubectl exec -it vault-0 -- vault operator unseal <unseal-key-2>
+    kubectl exec -it vault-0 -- vault operator unseal <unseal-key-3>
+    ```
+
+3. Login to Vault using the initial root token generated in step two:
+    ```sh
+    kubectl exec -it vault-0 -- vault login <initial-root-token>
+    ```
+
+4. Enable the the KV secrets engine at `secret/`:
+    ```sh
+    kubectl exec -it vault-0 -- vault secrets enable -path=secret kv-v2
+    ```
+
+## Configure Vault for Kubernetes authentication
+
+1. Enable Kubernetes authentication:
+    ```sh
+    kubectl exec -it vault-0 -- vault auth enable kubernetes
+    ```
+
+2. Configure Vault with the Kubernetes API:
+    ```sh
+    kubectl exec -it vault-0 -- sh
```

Comment:
consider separating in two commands
one with `kubectl exec -it vault-0 -- sh`
one with what gets inside the shell
simply because in practice, users will command/paste step by step, not all of this chained in one